sdks/wasm/Mono.WebAssembly.DebuggerProxy/DebugStore.cs

```diff
@@ -356,11 +387,11 @@ public AssemblyInfo (string url, byte[] assembly, byte[] pdb)
 			try {
 				Url = url;
 				ReaderParameters rp = new ReaderParameters (/*ReadingMode.Immediate*/);
-				rp.ReadSymbols = true;
-				rp.SymbolReaderProvider = new PdbReaderProvider ();
-				if (pdb != null)
+				if (pdb != null) {
+					rp.ReadSymbols = true;
```

Comment:
This breaks embedded pdbs, remove the logging instead

Reply:
So if an assembly have a missing pdb file, we actually fail later:
dbug: WebAssembly.Net.Debugging.DevToolsProxy[0]
      Failed to load http://localhost:9400/managed/netstandard.dll (Argument null or empty (Parameter 'fileName'))